vmdb/db/migrate/20150326154800_add_application_install_time.rb

```diff
@@ -0,0 +1,5 @@
+class AddApplicationInstallTime < ActiveRecord::Migration
+  def change
+    add_column :guest_application, :installtime, :datetime
```

Comment:
- Prefer the column name to be `install_time`
- Prefer `timestamp` for the column type over `datetime`